src/main/resources/static/stylesheet.css

```
@@ -24,27 +69,114 @@ body {
 	font-size: 25px;
 	border-bottom: solid 3px #d3d3d3;
 	margin-top: 30px;
+	margin-bottom : 30px;
 }
 
-table th{
-  padding : 20px;
+.align-middle {
+	box-sizing: border-box;
 }
 
-table td{
-  padding : 20px;
+table {
+	margin-left: auto;
+    margin-right: auto;
+    
 }
```

```suggestion
table {
    margin-left: auto;
    margin-right: auto;
}
```
{}の中でインデントばらばらは見づらいので気をつけていきましょう。